tools-v2/pkg/cli/command/curvebs/check/copyset/copyset.go

```diff
@@ -352,7 +352,7 @@ func CheckCopysets(caller *cobra.Command) (*map[uint64]cobrautil.ClUSTER_HEALTH_
 	return cCmd.Key2Health, cmderror.Success()
 }
 
-func GetCopysetsStatus(caller *cobra.Command) (*tablewriter.Table, *cmderror.CmdError) {
+func GetCopysetsStatus(caller *cobra.Command) (*interface{}, *tablewriter.Table, *cmderror.CmdError) {
```

Comment:
Harmonize with other methods:
```diff
- func GetCopysetsStatus(caller *cobra.Command) (*interface{}, *tablewriter.Table, *cmderror.CmdError) {
+ func GetCopysetsStatus(caller *cobra.Command) (*interface{}, *cmderror.CmdError) {
```

Reply:
The func is invoked at  https://github.com/opencurve/curve/blob/e770d4aae4395237efdee829235ee3569bef4a2d/tools-v2/pkg/cli/command/curvebs/status/copyset/copyset.go#L75
and it seems that there is no need to unify

Reply:
When we use a sub-command, just return a `*interface{}` is better(let user to parse it and put it into a table)?

Reply:
As we above discussion, return a table object seem not a good solution.
> > What does mean? cmd.Result is all rows not a table obj and can not render it directly.
> 
> The table object is a list, and it is very ugly to match.

Reply:
done